src/main/java/org/folio/circulation/domain/FeeFine.java

```diff
@@ -39,4 +47,8 @@ public String getId() {
   public String getFeeFineType() {
     return feeFineType;
   }
+
+  public static Set<String> lostItemFeeTypes() {
+    return LOST_TEM_FEE_TYPES;
+  }
```

Comment:
Please can this static method be moved to the top of the class where it is more easily discovered